tests/tests_integration/test_api/test_datapoints.py

```diff
@@ -313,6 +313,27 @@ def parametrized_values_uniform_index_fails(testrun_uid):
         )
 
 
+@pytest.fixture(scope="module")
+def timeseries_degree_c_0_100(cognite_client: CogniteClient):
+    timeseries = TimeSeries(
+        external_id="test_retrieve_datapoints_in_target_unit",
+        name="test_retrieve_datapoints_in_target_unit",
+        is_string=False,
+        unit_external_id="temperature:deg_c",
+    )
+    created_timeseries: TimeSeries | None = None
+    cognite_client.time_series.delete(external_id=timeseries.external_id, ignore_unknown_ids=True)
+
+    try:
+        created_timeseries = cognite_client.time_series.create(timeseries)
```

Comment:
To avoid windows-ubuntu-runner-race-conditions, let's do `upsert` here, without deleting after.